deps/build.jl

```diff
@@ -1,12 +1,18 @@
-using Conda
+#using Conda
 
-ENV["PYTHON"] = ""  # Configure PyCall.jl to use Conda.jl's Python
+ENV["PYTHON"] = "/usr/bin/python3.6"
```

Comment:
This concerns me a bit, because it won't work if the local python installation is in a different directory. Is there a reason why we can't use Conda.jl's Python?